api/jobs/ping.js

```diff
@@ -57,7 +57,7 @@ class PingJob extends job.Job {
                                     timeout: 4E3,
```

Comment:
On the https://github.com/Countly/countly-server/pull/4460/files#diff-093d3ee8e3c1f466c82d6f8f2a0fa1ff3b4b507a6e3f123b2d6d6cc7ae790cd2R50 We need to check if the frontend country_tracking setting is enabled and not check config file anymore